src/components/base-address/package.json

```diff
@@ -11,7 +11,7 @@
     "vuelidate": "^0.7.4"
```

Comment:
While you're here, can you please bump this  up to `^0.7.7`? (same as the hoisted version used by SB)

Reply:
Obsolete comment. (It's actually an older version we need 😛 )